optimize/self-managed/optimize-deployment/migration-update/camunda-8/3.13_8.5-to-8.6.md

```diff
@@ -1,13 +1,13 @@
 ---
-id: 3.13_8.5-to-3.14_8.6
-title: "Update notes (8.5/3.13 to 8.6/3.14)"
+id: 3.13_8.5-to-8.6
+title: "Update notes (8.5/3.13 to 8.6)"
 ---
 
 :::note Heads up!
-To update Optimize to version 8.6/3.14, perform the steps in the [migration and update instructions](./instructions.md).
+To update Optimize to version 8.6, perform the steps in the [migration and update instructions](./instructions.md).
 :::
 
-The update to 8.6/3.14 can be performed from any 8.5/3.13 release.
+The update to 8.6 can be performed from any 8.5/3.13 release.
```

Comment:
I am just wondering if we are somehow steering users into making an error here. If a C7 user on 3.13 sees this, they might incorrectly feel like they need to update to 8.6. If they do this, they may end up in a very bad and unrecoverable DB state. I am considering whether it would make sense to be stricter and simply refer to these pages by their C8 version. Wdyt?

Reply:
Totally agree with you, i think i just missed removing the C7 version here.

Reply:
Totally agree with you, I think I just missed this.